src/mainboard/pcengines/apu2/acpi/routing.asl

```diff
@@ -70,6 +71,7 @@ Name(PR0, Package(){
 Name(APR0, Package(){
 	/* NB devices in APIC mode */
 	/* Bus 0, Dev 0 - F15 Host Controller */
+	Package(){0x0000FFFF, 0, 0, 28 },
```

Comment:
@thillux int28 corresponds to INTE for PIC mode, and INTA corresponds to int24. it should be fixed either here or line above. How did You make choice for int28? Is it an arbitrary value?

Reply:
It was an rather arbitrary value, as Linux seems to select interrupts without using this number. Linux seems to only use the routing information from traversing the nested packages.